packages/larva-tokens/style-guides/deadline.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
This looks like an extra file generated - can you delete it? Sometimes the script does that, I'm not sure why.